setup.py

```diff
@@ -8,4 +8,7 @@
     test_suite='nose.collector',
     platforms=['any'],
     zip_safe=False,
+    extras_require={
```

Comment:
The difference between extras and env markers is that installing extras is a **user**'s choice (like feature flags, and extra names may even be treated as a kind of public API), on the other hand environmental markers force requirements to be installed under certain conditions pre-defined by **developer**, it's not user's choice.

Environment markers constraints are useful in case you need some extra library to be installed in certain env type. i.e. win32 may require some shims or Python 3.3 100% requires installing asyncio from PYPI, since it's not shipped as a part of its stdlib.

I'd say leave it under extras w/o env markers, but just move to `setup.py`.